site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -711,9 +711,14 @@ All assets must be declared as [Web Accessible Resources][manifest-war] in the `
 
 ## Stay secure {: #security }
 
+{% Aside 'warning' %}
```

Comment:
I don't think that we need to explicitly call this out. In particular in this file as it is not service worker related.